src/games/googlegossip/style.css

```diff
@@ -48,7 +48,7 @@
   color: white;
   border-color: white;
 }
-input {
+.gg-input {
```

Comment:
Ditto merge conflict comment regarding [this commit](https://github.com/jtessler/studio-multiplayer-game/commit/e6b793771eaf1f630e7c4a1969885351ef0045a6).